sections/footer.liquid

```diff
@@ -72,6 +72,22 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'image' -%}
+                  <div class="footer-block__details-content footer-block-image{% if block.settings.alignment != 'left' %} {{ block.settings.alignment }}{% endif %}">
+                    {%- if block.settings.image != blank -%}
+                      <img
```

Comment:
So the only reason I didn't have it here is because it didn't seem to make a big difference. Since the max-width can be up to `200px` so up to `400px` on high res screens. Not sure if it's worth the logic for a 200px difference in terms of image definition. What do you think ? I could add something like that though if you think it's beneficial: 

```
{%- if block.settings.image.width >= 200 -%}{{ block.settings.image | img_url: '200x' }} 200w,{%- endif -%}
{%- if block.settings.image.width >= 400 -%}{{ block.settings.image | img_url: '400x' }} 400w{%- endif -%}
```

Reply:
I agree the difference wouldn't be that noticeable, but loading the correct image size would be good as a best practice. I think we could leverage the `block.settings.image_width` setting to set the resolution of the image (similar to header logo):

``` html
{%- assign image_size = block.settings.image_width | append: 'x' -%}
<img srcset="{{ block.settings.image | img_url: image_size }} 1x, {{ block.settings.image | img_url: image_size, scale: 2 }} 2x"
  src="{{ block.settings.image | img_url: image_size }}"
  ...
>
```